### ISSUE TITLE
Can I add more than one main account?

### ISSUE THREAD

**Issue by FPSGuilherme** [Gunthersuper/steam-idle-bot#2]
I wonder if there is a way to add more than one main account, or if there is a way to remove this option, so that anyone can use the commands.

**Comment by Gunthersuper** [comment]
No, you can use only one main account rn. If you remove this, you wont be able to get access to the chat, I think it would be better for safety

**Comment by Zamjirou** [comment]
no wonder.. i cant use 2 main acc

**Comment by Zamjirou** [comment]
now my prob solved
